packages/blocks/end-flow/src/lib/actions/end-workflow.ts

```diff
@@ -14,6 +14,6 @@ export const endFlowAction = createAction({
     },
   },
   async run(ctx) {
-    return ctx.run.stop({ response: {} });
+    return ctx.run.stop({ response: {} }) ?? true;
```

Comment:
@rita-gorokhod like this?

Reply:
@bigfluffycookie seems like .stop is a void func, so you can do
```
ctx.run.stop({ response: {} });
return true;
```